### PR TITLE
V1

### PR DESCRIPTION
**Pull request by hai-le-di**

### CODE REVIEW

File: cinema/models.py [hunk 1]
```diff
@@ -134,9 +136,8 @@ def save(
         )
 
     def __str__(self):
-        return (
-            f"{str(self.movie_session)} (row: {self.row}, seat: {self.seat})"
-        )
+        return f"{str(self.movie_session)} " \
+               f"(row: {self.row}, seat: {self.seat})"
```
Comment (olena-hul) [inline cinema/models.py:140]: Why did you change this?

File: cinema/views.py [hunk 2]
```diff
@@ -127,6 +125,28 @@ def upload_image(self, request, pk=None):
...
+                description="Filter by actor id (ex. ?actors=1,3)",
+            ),
+            OpenApiParameter(
+                "genres",
+                type={"type": "list", "items": {"type": "number"}},
+                description="Filter by genre id (ex. ?genres=1,3)",
+            ),
+        ]
+    )
+    def list(self, request, *args, **kwargs):
```
Comment (olena-hul) [inline cinema/views.py:147]: Add annotation

File: cinema/views.py [hunk 3]
```diff
@@ -167,6 +186,23 @@ def get_serializer_class(self):
...
+        parameters=[
+            OpenApiParameter(
+                "movie",
+                type={"type": "int"},
+                description="Filter by movie id (ex. ?movie=1)",
+            ),
+            OpenApiParameter(
+                "data",
+                type={"type": "data"},
+                description="Filter by data (year-month-date)",
```
Comment (olena-hul) [inline cinema/views.py:199]: example missing

File: cinema/views.py [hunk 4]
```diff
@@ -167,6 +186,23 @@ def get_serializer_class(self):
...
+            ),
+        ]
+    )
+    def list(self, request, *args, **kwargs):
```
Comment (olena-hul) [inline cinema/views.py:203]: Add annotation

**Review by olena-hul — CHANGES_REQUESTED** [review]

File: cinema/models.py (hunk 1 above)
Comment (hai-le-di) [reply cinema/models.py:140]: flake8

**Review by Bohdan-Salamakha — APPROVED** [review]
